### ISSUE TITLE
SSH access suddenly fails?

### ISSUE THREAD

**Issue by asjustis** [ForkIssues/Tracker#1475]
Hey guys,

I was using Fork on Mac for a long while now and today randomly it stopped working giving me the access error below. I did not recall doing anything out of the ordinary (diff/commit/pull stuff).

I am using an SSH key generated via Fork and have it added in Github, and it used to work for a long time. Any idea what would be the cause? (I am on the latest 2.11.2 version) Or is this MS already doing things to Github? :D 

In the Fork -> Accounts menu it had a yellow account status: "cannot authenticate" or something, but I logged in again and gave needed (probably missing) permissions, and now it is green "Online", but the issue did not go away.

```
Pushing to github.com:<...private git repo I am invited to...>
ERROR: Repository not found.
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.


In a case you entered incorrect password, please
update it in Keychain Access application.
```

**Comment by DanPristupov** [comment]
Does this happen only with that particular repo or with all repos?

First of all, try to restart the computer. There is a rare problem in macOS, which blocks any interactions KeyChain access until restart.

If the problem persists, then check the SSH configuration dialog (Fork -> Configure SSH keys). Ensure that the correct SSH key is selected there.

**Comment by asjustis** [comment]
Thanks for quick reply Dan! 

Actually, this happened only with all repositories from Organization where I am an "outside collaborator". 

After following your suggestions, I noticed I couldn't use the SSH key from terminal as well for some weird reason. So I just generated a new SSH key, selected it in the Fork app and now it seems to be resolved. 

So it looks like it is not Fork issue, but something with git since it started happening out of the blue. Oh well, thanks for the help!
